conf/nutcracker.yml

```diff
@@ -65,3 +65,18 @@ omega:
   servers:
    - 127.0.0.1:11214:100000
    - 127.0.0.1:11215:1
+
+sigma:
+  listen: 127.0.0.1:22125
+  hash: fnv1a_64
+  distribution: ketama
+  auto_eject_hosts: false
+  redis: true
+  server_retry_timeout: 2000
+  server_failure_limit: 1
+  servers:
+    - 127.0.0.1:6379:1 server1
+    - 127.0.0.1:6380:1 server2
+  sentinels:
```

Comment:
There is actually a good reason to connect to multiple Sentinels. If the Sentinel you are connected to is on the minority side of a network partition, and the original master is in that partition, it could report it's "old" master. By querying all of the Sentinels you'd detect it and route to the majority master.